castle/cms/upgrades/upgrades_2_6_32.py

```diff
@@ -0,0 +1,10 @@
+from Products.CMFCore.utils import getToolByName
```

Comment:
A month or so back, I refactored how we do upgrades to save some code and files. For a "standard" upgrade like this, this file is now unnecessary, but we do need a line in castle/cms/upgrades/__init__.py for 2_6_32.